gtk2/fcitximcontext.cpp

```diff
@@ -430,7 +430,17 @@ static void fcitx_im_context_finalize(GObject *obj) {
     g_clear_pointer(&context->preedit_string, g_free);
     g_clear_pointer(&context->surrounding_text, g_free);
     g_clear_pointer(&context->attrlist, pango_attr_list_unref);
-    g_queue_clear_full(&context->gdk_events, (GDestroyNotify)gdk_event_free);
+    /* https://github.com/GNOME/glib/blob/main/glib/gqueue.c#L164
+     * Compatible with glib 2.5.58 < 2.60
+     * g_queue_clear_full(&context->gdk_events,
+     * (GDestroyNotify)gdk_event_free);*/
+    g_return_if_fail(&context->gdk_events != NULL);
```

Comment:
just think about it, how can &XX == null?

Reply:
When the gtk event is empty, it cannot enter here, so there is no need to judge, so these codes are useless